### ISSUE TITLE
Propose New Constitution or Guardrails Script Governance Action for an offchain discussion and submit it as a governance action

### ISSUE THREAD

**Issue by bosko-m** [IntersectMBO/govtool#2258]
* Enable New Constitution or Guardrails Script GA type to be discussed offchain through the proposal discussion forum section
   * Add option **Updates to the Constitution** to the dropdown when proposing new GA
   * Use **Treasury** action as a reference
      * Instead of two fields for rewards make the following ones:
         * **New constitution URL**
         * Checkbox toggle with the following copy: **Do you want to provide new guardrails script data?** Upon clicking it should show/hide the following two fields:
            * **Guardrails script URL**
            * **Guardrails script hash**
      * There should be a validation on the **New constitution URL** as it needs to check whether its able to fetch content and whether its a text file
      * **New constitution URL** should support ipfs links as well
      * Keep **title**, **abstract**, **motivation** and **rationale**
      * Keep optional section for providing additional linkes
   * Constitution type should also be adequately displayed on the page listing all proposals
      * It should have its own section
      * It should also be available in the filter dropdown

Notes:
* We should also change:
   * **Treasury** to **Treasury requests**
   * **Info** to **Info Action**
* **Submit as Governance Action** button should be disabled for this specific type, **Update to the Constitution**
   * Once done it should be deployed to Preview to gather community feedback
   * The button can then be enabled along with the solution to submit tx on chain which is the part voting pillar team should be responsible for

**Comment by bosko-m** [comment]
Description changed
@Ryun1 @nebojsact @l-br1

**Comment by l-br1** [comment]
separate idea > for discussion of this GA we could link to https://constitution.gov.tools/en/interim-constitution

**Comment by bosko-m** [comment]
@l-br1 

> please double check the name of this option in the dropdown as it should match the GA 'Update to the constitution'

If we are to respect full names according to the CIP-1694 it should be **New Constitution or Guardrails Script**
Its also displayed as a label on a listed proposal:

<img width="369" alt="Image" src="https://github.com/user-attachments/assets/ded61e94-bf93-4b44-a7a4-543678e63cc1" />

For example, **Treasury** is listed in the dropdown when you are choosing type and also as a label, while by CIP-1694 its full name is **Treasury Withdrawals**

Anyway, copy is really easy to be changed. 🙌

**Comment by l-br1** [comment]
Yes, the name on 1694 is not always self-explanatory. If you check on the voting page filter you have a more user-friendly name, for now we should align to that

**Comment by l-br1** [comment]
p.s. for treasury it would be better 'Treasury requests' or 'Treasury withdrawal requests'
